src/github/api/operations.rs

```diff
@@ -154,7 +169,7 @@ async fn update_branch(
             url.as_str(),
             Some(&serde_json::json!({
                 "sha": sha.as_ref(),
-                "force": force
+                "force": bool::from(force)
```

Comment:
Could you just match on the value here instead of implementing `From`? It's a small thing, and it doesn't really matter at  this point in the codebase, but it's not a general property of `ForcePush` that it can be converted into a bool in an unambiguous way. We could have some code in the future that needs to generate `true` for `ForcePush::No` and vice-versa, so it should be decided on a per call-site basis.